### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anders-wind**

### CODE REVIEW

**Comment by anders-wind** [comment]
TODO update first example with an `unwrap<T>`

File: README.md [hunk 1]
```diff
@@ -14,6 +14,16 @@
 
 === An easy to customize, strong type library with built in support for unit-like behavior ===
 
+#### What:
+- Easy support for stron{g|k} typing, with plenty of built-in skills to add functionality your own types.
```
Comment (casperkaae) [inline README.md:18]: - Easy support for stron{g|k} typing, with plenty of built-in skills to add functionality **to** your own types.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,16 @@
...
+- Automatically combine types with physics-like unit behavior.
+
+#### Why:
+- Strong types allow you to catch argument ordering mismatches.
+- Unit-like behavior allows you to use the type system to to verify the correctness of your implementations.
```
Comment (casperkaae) [inline README.md:23]: "to to" -> "to"

File: README.md [hunk 3]
```diff
@@ -217,7 +230,9 @@ target_link_libraries(
 )
 ```
 
-### Dependencies
+### Requirements
+A c++20 compatible compiler and standard library with concepts support.
+
 We depend on Boost's type_index package to get compile time generated ids for each type to be able to sort types for units (so we can compare types generated from different expressions).
 
 In the extensions subfolder we have added skills for common third party libraries: `fmt`, `absl` and `gtest`. Using any of these will ofcourse also require you to install those dependencies.
```
Comment (casperkaae) [inline README.md:238]: In the extensions subfolder we have added skills for common third party libraries: `fmt`, `absl` and `gtest`. Using any of these will ofcourse also require you to install those dependencies.
-->

In the extensions subfolder we have added skills for common third party libraries: `fmt`, `absl` and `gtest`. using these also requires the relevant third party libraries to be installed.

**Review by casperkaae — DISMISSED** [review]

**Review by casperkaae — APPROVED** [review]

**Pull request merged**